.gitattributes

```diff
@@ -0,0 +1 @@
+* -lf
```

Comment:
I've always used `* text=auto` syntax - what does this do differently?

Reply:
that's what e.g. react does: https://github.com/facebook/react/blob/a0d991fe6587ad1cd1a97230f62f82c7cb6b9a40/.gitattributes

Reply:
[gitattributes docs on *text*](https://git-scm.com/docs/gitattributes#_text):

> This attribute enables and controls end-of-line normalization. When a text file is normalized, its line endings are converted to LF in the repository.
> If you want to ensure that text files that any contributor introduces to the repository have their line endings normalized, you can set the text attribute to "auto" for all files.

[gitattributes docs on *text set to string value "auto"*](https://git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Settostringvalueauto)

> When text is set to "auto", the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin.
> When the file has been committed with CRLF, **no conversion is done**.

Another issue: If I understand correctly, `text=auto` doesn't handle the line ending format in the working directory.
For linting to pass, the line ending format in the working directory must be `LF` regardless of platform.

[gitattributes docs on *eol*](https://git-scm.com/docs/gitattributes#_eol)

> This attribute sets a specific line-ending style to be used in the working directory. It enables end-of-line conversion without any content checks, effectively setting the text attribute.

Reply:
From my reading using `* text=auto` is fine since no files are currently CRLF checked in (and if there are, we should convert them).

Reply:
auto is what GH has, so let's use that.